core/templates/dev/head/admin/Admin.js

```diff
@@ -23,6 +23,39 @@ oppia.controller('Admin', ['$scope', '$http', function($scope, $http) {
   $scope.adminTopicsCsvDownloadHandlerUrl = '/admintopicscsvdownloadhandler';
   $scope.configProperties = {};
 
+  $scope.activities = true;
```

Comment:
Can we have `$scope.currentTab = TAB_ACTIVITIES;`, where `TAB_ACTIVITIES` is the string 'activities', and just change the value of $scope.currentTab as needed?